examples/iterate.rs

```diff
@@ -64,3 +65,6 @@ fn main() {
         }
     }
 }
+
+#[cfg(not(any(target_os = "macos", target_os = "freebsd")))]
```

Comment:
Don't disable this test for Linux. Instead implement `struct Ctl` (with a string as member instead of Vec<c_int>) and the Ctl iterator that iterates over the contents of `/proc/sys`.